pages/about.html

```diff
@@ -4,9 +4,48 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="../styles/about-style.css" rel="stylesheet">
```

Comment:
You can actually import multiple stylesheets so you could have one that had all your shared styles and then a separate one per page (you mentioned wanting to DRY up your CSS):

```suggestion
  <link href="../styles/common-style.css" rel="stylesheet">
  <link href="../styles/about-style.css" rel="stylesheet">
```